packages/smooth_app/lib/main.dart

```diff
@@ -53,6 +53,7 @@ class _SmoothAppState extends State<SmoothApp> {
   late ProductPreferences _productPreferences;
   late LocalDatabase _localDatabase;
   late ThemeProvider _themeProvider;
+  late UserManagementProvider _userManagementProvider;
```

Comment:
Should be `final UserManagementProvider _userManagementProvider = UserManagementProvider();` instead.